pkg/crypto11/doc.go

```diff
@@ -0,0 +1,8 @@
+// package crypto11 provides a wrapper arround the plain pkcs11 Go wrapper,
+// which supports signing operations according to the crypto package for RSA
+// based keys.
+// It is extracted from https://github.com/thalesgroup/crypto11, which seems to
+// be not maintained anymore.
+// It enriches the low level signing operations offered by PKCS#11 to be compliant
+// to the interface offered by the crypto rsa package offered by Go.
```

Comment:
why did we extract it? what are the features required to be added?

Reply:
Why is explained in the comment.  I added the default signing parameter calculation as feature.